pynanoleaf/pynanoleaf.py

```diff
@@ -9,6 +9,8 @@ def _deepgetter(key):
     def get(data):
         for k in key:
             data = data[k]
+        if ('.'.join(key) == 'effects.select'):
```

Comment:
That is an awesome change, could you please do this in a separate PR? Makes it easier to review :)

Reply:
Hey, I'll chime in here too :)

I think it's better to keep this utility function free of any logic regarding specific fields. Instead write a regular property for this special case.

```
@property
def effectSelect(self):
    return _parseEffect(self['effects']['select'])
```

Reply:
Ok